charts/vineyard/values.yaml

```diff
@@ -93,10 +93,10 @@ tieredstore:
 
 volumes: []
 
-disabledPrometheus: false
+disablePrometheus: true
 
 fuse:
-  image: vineyardcloudnative/mount-vineyard-socket
+  image: vineyardcloudnative/vineyard-fluid-mount
```

Comment:
How about naming it to vineyard-fluid-fuse?

Reply:
Done.